instrumentation/jetty-httpclient/jetty-httpclient-9.0/library/src/main/java/io/opentelemetry/javaagent/instrumentation/jetty/httpclient/v9_0/JettyHttpClient9Tracer.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.jetty.httpclient.v9_0;
+
+import static io.opentelemetry.javaagent.instrumentation.jetty.httpclient.v9_0.RequestHeaderInjectorAdapter.SETTER;
+
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.context.propagation.TextMapSetter;
+import io.opentelemetry.instrumentation.api.tracer.HttpClientTracer;
+import io.opentelemetry.instrumentation.api.tracer.net.NetPeerAttributes;
+import java.net.URI;
+import java.net.URISyntaxException;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.eclipse.jetty.client.api.Request;
+import org.eclipse.jetty.client.api.Response;
+import org.eclipse.jetty.http.HttpFields;
+
+public class JettyHttpClient9Tracer extends HttpClientTracer<Request, Request, Response> {
+  private static final JettyHttpClient9Tracer TRACER = new JettyHttpClient9Tracer();
+
+  public static final JettyHttpClient9Tracer tracer() {
+    return TRACER;
+  }
+
+  private JettyHttpClient9Tracer() {
+    super(NetPeerAttributes.INSTANCE);
+  }
+
+  @Override
+  protected String getInstrumentationName() {
+    return "io.opentelemetry.javaagent.jetty-httpclient-9.0";
+  }
+
+  @Override
+  protected String method(Request request) {
+    return request.getMethod();
+  }
+
+  @Override
+  protected @Nullable URI url(Request request) throws URISyntaxException {
```

Comment:
```suggestion
  @Nullable 
  protected URI url(Request request) throws URISyntaxException {
```
seems to be the preferred way.

Reply:
Thanks for the suggestion,  will do